requirements.txt

```diff
@@ -1 +1,8 @@
 pylint==1.8.4
+django>=2.0.0
```

Comment:
We can save this for a story around using pipenv, but this would be converted to a `Pipfile` then.